bot.py

```diff
@@ -52,4 +54,16 @@ async def bruh(ctx):
                              '<:dangBRUH:661420374390603776>']))
 
 
+@bot.command()
+async def schedule(ctx):
+    """Returns the most current schedule"""
+    async with aiohttp.ClientSession() as session:
```

Comment:
Great you used a context manager here and below. This should ensure resources are freed up when the blocks of code are done executing.